### DISCUSSION TITLE
paddlespeech asr 识别 问题

### DISCUSSION THREAD

**Discussion by jkluo** [PaddlePaddle/PaddleSpeech#1780] [Q&A]
1.生产环境机器部署一般选用什么类型的机器呢，我 4核8G 的gpu机器，只能识别出60s以下的音频。这个是什么原因呢，超过60s，直接返回空了。

![image](https://user-images.githubusercontent.com/12638923/165036603-aa68c25c-a6c8-4f4e-87dc-282d43d4d773.png)

2. 一个转译任务，就占用一核的cpu，整体cpu使用率很高，怎么优化呢
![image](https://user-images.githubusercontent.com/12638923/165036308-59abc7e3-0812-4f8b-a5b4-7de455e68aec.png)

**Comment by zh794390558** [comment]
input限制问题可以关注下issue。

**Reply by Dewey-Ding** [reply]
> input限制问题可以关注下issue。

@zh794390558   对于第二个问题，一个任务使用一核CPU  是否有优化方案
